pkg/doc/verifiable/support_test.go

```diff
@@ -354,3 +356,106 @@ func createVCWithTwoLinkedDataProofs() (*Credential, PublicKeyFetcher) {
 		panic("invalid keyID")
 	}
 }
+
+//nolint:gochecknoglobals
+var testDocumentLoader = createTestJSONLDDocumentLoader()
+
+func createTestJSONLDDocumentLoader() *ld.CachingDocumentLoader {
+	fmt.Printf("Create test document loader!\n")
+
+	loader := CachingJSONLDLoader()
+
+	const exampleJSONLDContext = `
+{
```

Comment:
it would be nice to move these test contexts into files under the `testdata` directory.

Reply:
done!